client/src/components/Workflow/Editor/Index.vue

```diff
@@ -541,6 +555,40 @@ export default {
             const step = { ...this.steps[nodeId], annotation: newAnnotation };
             this.onUpdateStep(step);
         },
+        async onCreate() {
+            if (!this.name || !this.annotation) {
+                const response = "Must provide name and annotation before creation...";
```

Comment:
These are user-facing messages. We (usually) know exactly what happened and specifically here we are asking user to correct. I believe something like `Please provide a name for your workflow.` and avoiding using ellipsis throughout the status messages (to create certainty instead) would be an improvement.